test/browser/run-test.sh

```diff
@@ -190,8 +190,9 @@ if [ "$PLAN" = "storage-basic" ]; then
             "
     fi
 
-    if [ "${TEST_OS%bootc}" != "$TEST_OS" ]; then
-        # no swap on TF bootc
+    if [ "$TEST_OS" = "centos-9-bootc" ]; then
+        # FIXME: pcp is installed, but not working on CentOS 9 (10 is fine)
```

Comment:
How does pcp break thje swap test? :-)

Reply:
swap usage metrics is pcp only, or at least our code only gets it from there.